Dockerfile

```diff
@@ -1,16 +1,24 @@
-FROM openjdk:alpine
+FROM openjdk:11-jre-slim-buster as stage0
+LABEL snp-multi-stage="intermediate"
+LABEL snp-multi-stage-id="61fb5002-8ada-43dc-928a-370d550c18da"
 WORKDIR /opt/docker
-ADD target/docker/stage/opt /opt
-RUN adduser -D -u 1000 drt
+COPY target/docker/stage/2/opt /2/opt
+COPY target/docker/stage/4/opt /4/opt
+USER root
+RUN ["chmod", "-R", "u=rX,g=rX", "/2/opt/docker"]
+RUN ["chmod", "-R", "u=rX,g=rX", "/4/opt/docker"]
+RUN ["chmod", "u+x,g+x", "/4/opt/docker/bin/drt-cirium"]
 
-RUN ["chown", "-R", "1000:1000", "."]
-
-RUN apk --update add bash less curl
-RUN rm -rf /var/cache/apk/*
+FROM openjdk:11-jre-slim-buster as mainstage
+USER root
+RUN id -u drt 1>/dev/null 2>&1 || (( getent group 0 1>/dev/null 2>&1 || ( type groupadd 1>/dev/null 2>&1 && groupadd -g 0 root || addgroup -g 0 -S root )) && ( type useradd 1>/dev/null 2>&1 && useradd --system --create-home --uid 1001 --gid 0 drt || adduser -S -u 1001 -G root drt ))
+WORKDIR /opt/docker
+COPY --from=stage0 --chown=drt:root /2/opt/docker /opt/docker
+COPY --from=stage0 --chown=drt:root /4/opt/docker /opt/docker
 
 RUN mkdir -p /var/data
-RUN chown 1000:1000 -R /var/data
-
-USER 1000
+RUN chown 1001:1001 -R /var/data
 
-ENTRYPOINT ["bin/drt-cirium"]
+USER 1001:0
```

Comment:
I guess this same as USER 1001

Reply:
This line is generated by `sbt docker:stage` so I guess it's just a docker way of specifying the user